helm/ovn-kubernetes/values-no-ic.yaml

```diff
@@ -147,6 +147,14 @@ global:
     tag: master
     # -- Image pull policy
     pullPolicy: IfNotPresent
+  # -- Image for DPUs
```

Comment:
NIT: Do we also need this in [values-multi-node-zone.yaml](https://github.com/ovn-org/ovn-kubernetes/blob/master/helm/ovn-kubernetes/values-multi-node-zone.yaml) and [values-single-node-zone.yaml](https://github.com/ovn-org/ovn-kubernetes/blob/master/helm/ovn-kubernetes/values-single-node-zone.yaml) ?
https://github.com/ovn-org/ovn-kubernetes/tree/master/helm/ovn-kubernetes

Reply:
I believe this fix does not include dpu support in IC mode.